### DISCUSSION TITLE
Bullet Library Licensing

### DISCUSSION THREAD

**Discussion by native-engine** [bulletphysics/bullet3#4041] [General]
Hello!

I am working on a game engine for my future game projects and am gradually getting closer to developing the first demo scene. And so I asked myself the question of licensing the libraries that I used to implement my game engine. The engine is developed exclusively for their own needs and only games made on it will be distributed. What conditions must be met in order not to violate the license to use this library?

To begin with, I indicated the use of this library in the user manual and in the future it will be so in all manuals. I also placed a copy of your license in the "documents / licenses" directory and would like to clarify if I took the correct text. If the text is different, please give the correct one.

Below I am attaching the license file and user manual for the future demo scene (one page is incomplete).
[Bullet License.txt](https://github.com/bulletphysics/bullet3/files/7500475/Bullet.License.txt)
[manual EN.pdf](https://github.com/bulletphysics/bullet3/files/7500477/manual.EN.pdf)

Sincerely, Daniil Petrov.

**Comment by native-engine** [comment]
On the page with logos on the texts of the titles, added hyperlinks to the sites.
[manual EN.pdf](https://github.com/bulletphysics/bullet3/files/7511529/manual.EN.pdf)

**Comment by erwincoumans** [comment]
Good luck with your game(s)! If you have a credit screen, some mention of Bullet Physics would be appreciated.
If you ship the source code, a link to the zlib license/manual will do.
